### ISSUE TITLE
Unable to trigger app context menu with keyboard only (MENU key)

### ISSUE THREAD

**Issue by megatux** [elementary/applications-menu#159]
Expect to open the context menu to do an action like "Add to dock" using the keyboard navigation + MENU-key only but the Search input context menu is activated instead.
Somewhat related issue #81 .

**Comment by cassidyjames** [comment]
To clarify, when an app is focused and the user presses their keyboard's [Menu key](https://en.wikipedia.org/wiki/Menu_key), the context menu opens in the search entry instead of on the focused app.
